### DISCUSSION TITLE
is there any limit on releases? hard limit on total size or count or anything?

### DISCUSSION THREAD

**Discussion by sku0x20** [community/community#40270] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i cannot find anything related to it after countless hours of searching. docs just say, limit on individual file. but they do not mention anything related to the number of releases or total size limit (mostly a bit worried about this). is it not there, or there are some hidden gotchas?

**Comment by ossanna16** [comment]
Hi there @siddhantkumarupmanyu and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
